SRTNet.cpp

```diff
@@ -345,6 +320,161 @@ bool SRTNet::startClient(const std::string& host,
 
     mClientContext = ctx;
 
+    mConfiguration.mLocalHost = localHost;
+    mConfiguration.mLocalPort = localPort;
+    mConfiguration.mRemoteHost = host;
+    mConfiguration.mRemotePort = port;
+    mConfiguration.mReorder = reorder;
+    mConfiguration.mLatency = latency;
+    mConfiguration.mOverhead = overhead;
+    mConfiguration.mMtu = mtu;
+    mConfiguration.mPeerIdleTimeout = peerIdleTimeout;
+    mConfiguration.mPsk = psk;
+
+    if (!createClientSocket()) {
+        SRT_LOGGER(true, LOGG_ERROR, "Failed to create caller socket");
+        return false;
+    }
+
+    // Get all remote addresses for connection
+    struct addrinfo hints = {0};
+    struct addrinfo* svr;
```

Comment:
Un-abbreviate these variable names to improve understandability

Reply:
renamed svr and hld to resolvedAddresses and resolvedAddress, this was old code which just got moved around in the file :)

Reply:
Good choices!